protocols/gossipsub/src/handler.rs

```diff
@@ -18,7 +18,7 @@
 // FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
-use crate::error::{HandlerError, ValidationError};
+use crate::error_priv::{HandlerError, ValidationError};
```

Comment:
```suggestion
use crate::{HandlerError, ValidationError};
```

I'd prefer if we import from the crate-root, same as what external users will do!

Reply:
changed all occurrences in gossipsub :)